nginx/supervisord.conf

```diff
@@ -0,0 +1,17 @@
+[supervisord]
```

Comment:
this is the Supervisord config file. It starts the nuxt app and then starts nginx